### ISSUE TITLE
Incorrect documentation example for Scenic.Components.text_field/3

### ISSUE THREAD

**Issue by kw7oe** [boydm/scenic#38]
I think the examples for [`Scenic.Components.text_field/3`](https://hexdocs.pm/scenic/Scenic.Components.html#text_field/3) are not updated. I might be wrong.

```elixir
graph
|> text_field( {"Sample Text", :text_id}, translate: {20,20} )

graph
|> text_field(
  {"", :pass_id, [type: :password, hint: "Enter password"]},
  translate: {20,20}
)
```

The updated examples should be as follow:
```elixir
graph
|> text_field( "Sample Text", id: :text_id, translate: {20,20} )

graph
|> text_field( "", id: :pass_id, type: :password, hint: "Enter password",
  translate: {20,20}
)
```

The examples code cause the app crash and restart continuously until we manually quit the process by `Ctrl-C`, which may also be an issue. If it is a concern, would it be better if we can impose limit on the restart attempt caused by such error, so we can avoid the supervisor to crash and restart indefinitely?

To reproduce the issue,
```
git clone https://github.com/kw7oe/scenic_demo.git
cd scenic_demo
mix deps.get
mix scenic.run
```

Point me to the right direction if I am wrong, since I am fairly inexperience. 

Thanks for the efforts in making this project. Really appreciate it.

**Comment by boydm** [comment]
Oh yes. good catch.

**Comment by boydm** [comment]
The restart limit is tricky when it happens during scene initialization. Is part of why I recommend building the graph at compile time instead of at run time.

If the scene crashes during any time, then the restart limits work fine.

**Comment by boydm** [comment]
Fixed in master. Will be published in 0.7.1 soonish.
